### DISCUSSION TITLE
what is the use of this syntax: export const fetchCache = 'force-no-store';

### DISCUSSION THREAD

**Discussion by abhishekmardiya** [vercel/next.js#65781] [Help] [answered]
### Summary

Without this syntax my web app is cached even though I've used  `export const dynamic = "force-dynamic";
`

**Comment by icyJoseph** [comment, marked as answer]
Hi,

The docs explain it here: https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#fetchcache

> fetchCache allows you to override the default cache option of all fetch requests in a layout or page.
